pkg/storage/pod_test.go

```diff
@@ -180,8 +186,8 @@ var _ = Describe("Pod storage", func() {
 		`), "metrics_server_storage_points")
 		Expect(err).NotTo(HaveOccurred())
 	})
-	It("should detect container restart and skip metric", func() {
-		s := NewStorage()
+	It("should detect container restart and return correct results for pod1", func() {
```

Comment:
```suggestion
	It("should detect container restart and return results based on window from start time", func() {
```
Let's be more concrete on what "correct" means